mesh/mesh.go

```diff
@@ -500,3 +500,63 @@ func (m *Mesh) GetATXs(atxIds []types.AtxId) (map[types.AtxId]*types.ActivationT
 	}
 	return atxs, mIds
 }
+
+// ActiveSetForLayerView - returns the active set size that matches the view of the contextually valid blocks in the provided layer
+func (m *Mesh) ActiveSetForLayerView(layer types.LayerID, layersPerEpoch uint16) (uint32, error) {
+
+	epoch := layer.GetEpoch(layersPerEpoch)
+	firstLayerOfPrevEpoch := types.LayerID(epoch-1) * types.LayerID(layersPerEpoch)
+
+	// build a map of all blocks between firstLayerOfPrevEpoch and our current layer
```

Comment:
comment is wrong, your putting the layer in the map

Reply:
Fixed